extensions/assets-manager/AssetsManagerEx.h

```diff
@@ -43,6 +43,15 @@
 
 NS_CC_EXT_BEGIN
 
+//! All files to be decompressed
+struct CompressedFilesInfoAsserts
+{
+    std::string     _fileName;
+    int             _nCompressIndex;
```

Comment:
i think it is better to rename to _compressIndex, `n` is used together with `m_`.